1.0/start-vault.sh

```diff
@@ -17,8 +15,12 @@ docker run -v $PWD/vault:/app \
            -e OUT_PATH=/app \
            -e VAULT_IP=$INSTANCE_IP \
            -e ENVIRONMENT=$ENV \
-           -e VAULT_TOKEN=$VAULT_TOKEN \
+           -e VAULT_TOKEN=unset \
```

Comment:
because of the way `dsde-toolbox` is set up now, this can't be missing or an empty string (despite not getting used in the rendering of these specific configs) because the `render-templates.sh` script will try and look for it in a default location instead.  This solution is kind of hacky but ensures that the vault token will be ignored, as we want in this case.